### PR TITLE
Add a second organization to seeds

### PR DESCRIPTION
**Pull request by Thrillberg**
### Describe your change in plain English.
This PR adds a second organization to the seeds file. It also renames a few variables and removes some unnecessary comments in the seedfile, which can easily get out of date.

This also adds AdopterApplications to the seeds file.

### Link to the issue
#44

### CODE REVIEW

**Review by edwinthinks — APPROVED** [review]

File: db/seeds.rb [hunk 1]
```diff
@@ -192,22 +214,18 @@
   city_town: 'Nonsense'
 )
 
-# create 10 pets
-
-10.times  do
+10.times do
   Pet.create!(
-    organization_id: 1,
+    organization: [@organization_one, @organization_two].sample,
```
Comment (garyhtou) [inline db/seeds.rb:219]: What are your thoughts on sampling from all Orgs? (less code to update in the future)

```suggestion
    organization: Organization.all.sample,
```

**Review by garyhtou — APPROVED** [review]
looks great!
